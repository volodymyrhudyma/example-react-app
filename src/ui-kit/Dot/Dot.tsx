import React from 'react';
import { Wrapper } from './styled/Dot.styled';

const Dot = () => <Wrapper />;

export default Dot;
