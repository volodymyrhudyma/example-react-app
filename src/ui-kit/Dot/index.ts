import Component from './Dot';

export default Component;
