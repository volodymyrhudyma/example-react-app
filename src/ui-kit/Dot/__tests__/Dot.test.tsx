import React from 'react';
import { shallow } from 'enzyme';
import Dot from '../Dot';

describe('Dot component', () => {
  it('should render correctly', () => {
    const component = shallow(<Dot />);
    expect(component).toMatchSnapshot();
  });
});
