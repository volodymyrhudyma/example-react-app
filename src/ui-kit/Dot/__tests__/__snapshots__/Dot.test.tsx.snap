// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dot component should render correctly 1`] = `<styled.div />`;
